addons/binding/org.openhab.binding.amazondashbutton/README.md

```diff
@@ -1,87 +1,120 @@
 ```
 
-If you need mulitple capabilities (like "cap_net_bind_service" for the Network binding), you have to add them like this :
+If you need mulitple capabilities (like "cap_net_bind_service" for the Network binding), you have to add them like this:
 
 ```shell
-sudo setcap 'cap_net_raw,cap_net_admin=+eip cap_net_bind_service=+ep' `realpath /usr/bin/java`
+sudo setcap 'cap_net_raw,cap_net_admin=+eip cap_net_bind_service=+ep' $(realpath /usr/bin/java)
 ```
 
-#### Other Operating Systems
+### Installing WinPcap on Windows
+
+On a Windows system there are two options to go with.
+
+1. The preferred solution is [WinPcap](https://www.winpcap.org) if your network interface is supported.
+2. An alternative option is [npcap](https://github.com/nmap/npcap) with the settings "WinPcap 4.1.3 compatibility" and "Raw 802.11 Packet Capture"
+
+### Installing libpcap on Other Operating Systems
+
+The installation methods might differ.
+A few known operating systems are:
 
 | Operating System | Command                     |
 |:-----------------|:----------------------------|
 | CentOS           | `yum install libpcap-devel` |
 | Mac              | `brew install libpcap`      |
-| Windows          | `choco install winpcap`     |
-
 
 ## Setup Dash Button
 
-Setting up your Dash button is as simple as following the instructions provided by Amazon **EXCEPT FOR THE LAST STEP**. Just follow the instructions to set it up in their mobile app. When you get to the step where it asks you to pick which product you want to map it to, just quit the setup process.
+Setting up your Dash Button is as simple as following the instructions provided by [Amazon](https://www.amazon.com/Dash-Buttons/b?node=10667898011) **EXCEPT FOR THE LAST STEP**.
+Follow the instructions to set up the Dash Button in their mobile app.
+When you get to the step where it asks you to pick which product you want to map it to, just quit the setup process.
 
-## Block Internet access for the Dash Button
+{: #no-phonehome}
+## Preventing Communication with Amazon Servers
 
-Completely deny internet access for the Amazon Dash Button in your router. You need to find out the Dash button's IP address first of all. This is not explained as this job depends on your environment.
+Be aware:
+Every time a Dash Button is pressed a request will be sent to the Amazon servers.
+If no product was configured for the Button, a notification will be presented by the Amazon app on your smartphone.
 
+To prevent the Dash Button from contacting the Amazon Servers, block Internet access for the device.
+Please refer to the documentation of your network's router for details.
+If your network doesn't provide that option, you can at least deal with the notifications by either uninstalling the Amazon app or disabling notifications for it (possible on most smartphone OSs).
 
-## Supported Things
+Preventing the communication with the Amazon servers or the Amazon app is **not** necessary to integrate the Dash Button in openHAB.
```

Comment:
You mentioned that we should warn others of the battery usage when disconnecting it. Might be worth adding that statement here?